src/widget/wstemlabel.cpp

```diff
@@ -0,0 +1,69 @@
+#include "wstemlabel.h"
+
+#include "moc_wstemlabel.cpp"
+#include "util/logger.h"
+
+const mixxx::Logger kLogger("WStemLabel");
+
+WStemLabel::WStemLabel(QWidget* pParent)
+        : WLabel(pParent),
+          m_stemInfo(QString(), QColor()),
+          m_stemNo(0) {
+}
+
+void WStemLabel::setup(const QDomNode& node, const SkinContext& context) {
+    m_stemNo = context.selectInt(node, "StemNum");
+
+    VERIFY_OR_DEBUG_ASSERT(m_stemNo >= 1 && m_stemNo <= mixxx::kMaxSupportedStems) {
+        kLogger.warning() << "Skin warning: StemNum is out of range. It should "
+                             "be between 1 and "
+                          << mixxx::kMaxSupportedStems;
```

Comment:
A mixxx::logger is nice, but here we'd actually use the `SKIN_WARNING` macro IMO